openlibrary/components/MergeUI/MergeTable.vue

```diff
@@ -103,34 +62,36 @@ function fetchRecord(olid) {
     const endpoint = `/${type}/${olid}.json`;
     // FIXME Fetch from prod openlibrary.org, otherwise it's outdated
     const url = location.host.endsWith('.openlibrary.org') ? `https://openlibrary.org${endpoint}` : endpoint;
-    return fetch(url).then(r => r.json());
+    return fetch(url).then(r => r.json()).then(json => {json.olid = olid; return json;});
```

Comment:
Oooh, I think it would be better to use key for the records. In some ideal future universe, we'll want to be able to type these api responses to be the same as the objects defined in https://github.com/internetarchive/openlibrary-client/tree/master/olclient/schemata . Also in the past, modifying records like this often results in inadvertently saving dev-only fields into our db, so best to keep to a minimum where possible!

Reply:
That's a good point — I hadn't considered the problem of saving the extra property in the database. I should fix that before we have people mess around with the new version.

The problem exists in the current code because full keys aren't available when the page loads, only OLIDs. Maybe we don't need to populate any rows until the full record set loads -- I'll look into it.

Reply:
Turns out there's no harm in leaving the selection array and the master_key empty until the record data loads. I assumed you had those prepopulating for some reason that I didn't understand. Sometimes I overthink things. :)

I've pushed a fix that rekeys the rows on the actually record key. If we could get that on testing, it's probably safe to let people play around with.